WearOAuth/build.gradle

```diff
@@ -25,8 +25,8 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.1'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath 'com.android.tools.build:gradle:4.2.2'
+        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.21'
```

Comment:
Nit: I think we should update the `kotlin_version` property above, since that's used elsewhere as well.